### ISSUE TITLE
JSON send

### ISSUE THREAD

**Issue by JeffreyDavidsz** [bitfocus/companion-module-eventmaster#20]
Hi Guys, can someone with a actual machine send me 2 or 3 examples of the data that's being send from E2/S3 when subscribing to it?
I'm looking for the JSON format where inputs are live (for Tally use), so I think I only need to subscribe to SourceChanged, BGSourceChanged. From this companion module you can use the function;

`case 'subscribe':
			log('info', 'subscribe to localhost');

			if (self.eventmaster !== undefined) {
				self.eventmaster.subscribe('127.0.0.1', 10001, ["SourceChanged","BGSourceChanged"], function(obj, res) {
					debug('subscribe response', res);
				}).on('error', function(err) {
					log('error','EventMaster Error: '+ err);
				});
			}
			break;`

notificationTypes > an array of notifications to which a user wants to subscribe.
ScreenDestChanged
AUXDestChanged
FrameChanged
NativeRateChanged
InputCfgChanged
SourceChanged
BGSourceChanged
PresetChanged
StillChanged
OutputCfgChanged

Example;
–	{"params": {"hostname" : "192.168.247.131", "port": "3000", "notification" : ["SourceChanged", "BGSourceChanged"]}, "method":"subscribe", "id":"1234", "jsonrpc":"2.0"}

Maybe @gerrylusk can help?

**Comment by gerrylusk** [comment]
Well, it doesn't seem to send something helpful. Or I haven't figured it out yet.
I subscribed to AUXDestChanged and it sent me a blurb of web-formatted JSON back when an AUX changed. Since it's an HTTP protocol, the connection should close after each message. And I seemed to have to re-subscribe each time it sent me a message. 

But I don't think I can subscribe to a particular AUX or Source. It only told me that a change happened, not what the change was. So I'm not sure how to get a tally system out of this except that you wouldn't have to poll the sources regularly, only when you received a notification (the QMSK tally system seems to poll a couple times per second).

This is what I got every time I switched any source to an AUX:
`{"result":{"method":"notification","notificationType":"AUXDestChanged","change":{"add":[],"remove":[],"update":[0,1]}}}`
